bridgenode/flatfileworker.go

```diff
@@ -60,10 +61,12 @@ type flatFileState struct {
 func flatFileWorker(
 	proofChan chan btcacc.UData,
 	ttlResultChan chan ttlResultBlock,
+	undoChan chan accumulator.UndoBlock,
 	utreeDir utreeDir,
 	fileWait *sync.WaitGroup) {
 
 	var ff flatFileState
+	var uf flatFileState
```

Comment:
Could we move this declaration to line 90 since it's first used in line 91-92? It'll be easier to read code if the declaration is just before it's actual usage.